### PR TITLE
add gzip request transormer

### PR DESCRIPTION
**Pull request by jensdev**

### CODE REVIEW

**Comment by jensdev** [comment]
Closes #242

**Comment by jensdev** [comment]
Needs service change, as it is currently giving CORS issue
image

**Review by galmail — APPROVED** [review]

**Comment by jensdev** [comment]
At the moment this is not working when calling from Node. Only in the Browser. Will have another look asap

File: src/services/data/documents.ts [hunk 1]
```diff
@@ -54,8 +54,17 @@ export default (client, httpAuth: HttpInstance) => ({
...
-    return (await client.post(httpAuth, `/${schemaId}/documents`, requestBody))
-      .data;
+    return (
+      await client.post(
+        httpAuth,
+        `/${schemaId}/documents`,
+        requestBody,
+        {},
+        {
+          gzip: true,
```
Comment (galmail) [inline src/services/data/documents.ts:64]: Do we always want to apply it? or can it be optional?

**Review by galmail — APPROVED** [review]
looks good, just left a question

**Review by galmail — APPROVED** [review]

**Pull request merged**